react/ProductList.tsx

```diff
@@ -6,7 +6,7 @@ import { getBaseUrl } from './modules/baseUrl'
 
 interface Product {
   productName: string
-  linkText: string
+  link: string
```

Comment:
Instead of renaming the property in the type definition, add a new one. It makes it more accurate, and also doesn't break the types API, for any app that actually calls this function.